pkg/azure/minter.go

```diff
@@ -137,10 +136,13 @@ func (credMinter *AzureCredentialsMinter) CreateOrGetServicePrincipal(ctx contex
 	case 0:
 		credMinter.logger.Infof("Creating service principal for AAD application %q", appID)
 		var servicePrincipal *graphrbac.ServicePrincipal
+		ownedTag := fmt.Sprintf("kubernetes.io_cluster.%s=owned", infraName)
```

Comment:
the separation in this tag is wierd.. `kubernetes.io` and `cluster` are separate by `_` while the `cluster` and `infraname` are separated by `.` ??

Reply:
@abhinavdahiya i was just trying to follow the existing tagging i see on Azure resources after install:
```
[jdiaz@minigoomba ~]$ az group show --name jdiaz-az-48vvg-rg | jq -r .tags
{
  "kubernetes.io_cluster.jdiaz-az-48vvg": "owned"
}
```

Reply:
I vote for consistency over correctness :) https://github.com/openshift/installer/blob/master/data/data/azure/main.tf#L4